openfisca_us/variables/contrib/ubi_center/basic_income/basic_income.py

```diff
@@ -19,4 +19,5 @@ def formula(person, period, parameters):
             tax_unit_phase_out / tax_unit_basic_income,
             0,
         )
-        return basic_income * (1 - percent_reduction)
+        eligible = tax_unit("basic_income_eligible", period)
+        return basic_income * (1 - percent_reduction) * eligible
```

Comment:
We could use `defined_for = "basic_income_eligible" here and remove this, though I think since there's practically zero performance benefit it's better to use pure OpenFisca formulas.

Reply:
Ah thanks, I switched to `defined_for` for consistency, since it'll improve performance in other cases to adopt the `defined_for = "*_eligible"` pattern.